cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/AwsClusterSettingsStep.tsx

```diff
@@ -16,42 +16,16 @@ import { ClusterName, clusterNameValidation } from '../../../shared/components/F
 import { FormAction } from '../../../shared/types/types';
 import { getResource } from '../../../views/providers/aws/AwsResources.reducer';
 import { INPUT_CHANGE } from '../../../state-management/actions/Form.actions';
-import {
-    NodeProfile,
-    NodeInstanceType,
-    nodeInstanceTypeValidation,
-} from '../../../shared/components/FormInputComponents/NodeProfile/NodeProfile';
+import { NodeProfile, nodeInstanceTypeValidation } from '../../../shared/components/FormInputComponents/NodeProfile/NodeProfile';
```

Comment:
Last suggestion, please change nodeInstanceTypeValidation to nodeProfileValidation. I know that is code you didn't write but would be good to change this now.

Reply:
Yes sir. Did not notice this previously. Thanks